pkg/v3/postprocessors/retry.go

```diff
@@ -29,7 +29,10 @@ func (p *retryablePostProcessor) PostProcess(_ context.Context, results []ocr2ke
 	retryable := 0
 	for i, res := range results {
 		if res.PipelineExecutionState != 0 && res.Retryable {
```

Comment:
nitpick - can we change the 0 to the enum? Isn't there something like `ExecutionState.Success`?

Reply:
Good callout, but I'd prefer to do it in a separate PR to avoid extra noise.. created AUTO-7018